src/settingsmanager.h

```diff
@@ -17,6 +17,7 @@ class SettingsManager : public QObject
     explicit SettingsManager(QObject *parent = nullptr);
     virtual ~SettingsManager() {};
 
+    QString getSettingsConfPath();
```

Comment:
This function doesn't need to be part of the `SettingsManager` public interface (or belong to `SettingsManager` at all). It can be defined in an unnamed namespace in `settingsmanager.cpp`.